ansible/roles/ocp4-workload-dil-serverless/tasks/provision_smartcity_edge_loop.yaml

```diff
@@ -0,0 +1,9 @@
+---
+
+- name: "Create smartcity edge workloads for all users"
+  include_tasks: 
+    file: provision_smartcity_edge.yaml
+  with_items:
+    - "{{ users }}"
+  loop_control:
```

Comment:
These loops can get too slow when there are many users, have you tested how long does it take to run for 100 users? We need to keep this under 10-30 mins total for the step.

Reply:
No, I didn't test for a large amount of users.
The thing I have dependencies (S3 buckets, AMQ Streams cluster) that need to be running before I can continue the deployment of the workloads, hence the loop.

Reply:
Can you give it a try with a 100 and time it?